src/generator/workers/src/output.ts

```diff
@@ -0,0 +1,193 @@
+// Generated Cloudflare Workers API
+// Version: 1.0
+// Project: test_project
+
+// IMPORTS
+
+// Import generated models
+import { Person } from './models';
+
+
+// TYPE DEFINITIONS
+
+type Handler = (...args: any[]) => Response;
+
+// ROUTER STRUCTURE (TRIE)
+
+// Trie-based router structure
+const router = {
```

Comment:
nit: you don't have to try to make tabs and spaces and so on for the generated code, does not need to look pretty. we could one line the whole thing and just run a typescript formatter on it if we wanted to